Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/Components/SchoolAndTrustInformation/default.cshtml

```diff
@@ -15,8 +15,13 @@
 	<govuk-summary-list-row name="name-of-trust" label="Trust name" key-width="@kw" asp-for="NameOfTrust" />
 	<govuk-summary-list-row name="trust-reference-number" label="Trust reference number" key-width="@kw" asp-for="TrustReferenceNumber" />
 	<govuk-summary-list-row name="project-recommendation" label="Recommendation" key-width="@kw" value="@Model.RecommendationForProject" asp-page="@Links.SchoolAndTrustInformationSection.ProjectRecommendation.Page" asp-route-id="@Model.Id" hidden-text="recommendation" />
-	<govuk-summary-list-row name="academy-order-required" label="Is an academy order (AO) required?" key-width="@kw" value="@Model.AcademyOrderRequired" asp-page="@Links.SchoolAndTrustInformationSection.AcademyOrderRequired.Page" asp-route-id="@Model.Id" hidden-text="academy order required" />
-   <govuk-summary-list-row if="@Model.IsDao" name="dao-pack-sent-date" label="Date directive academy order (DAO) pack sent" key-width="@kw" value="@Model.DaoPackSentDate" asp-page="@Links.SchoolAndTrustInformationSection.DaoPackSent.Page" asp-route-id="@Model.Id" />
+    <govuk-summary-list-row name="academy-order-required" label="Is an academy order (AO) required?" key-width="@kw" value="@Model.AcademyOrderRequired" asp-page="@Links.SchoolAndTrustInformationSection.AcademyOrderRequired.Page" asp-route-id="@Model.Id" hidden-text="academy order required" />
+    <govuk-summary-list-row if="@Model.IsDao" name="form-7-received" label="Has the Schools Notification Mailbox (SNM) received a Form 7?" key-width="@kw" value="@Model.Form7Received" asp-route-id="@Model.Id" asp-page="@Links.SchoolAndTrustInformationSection.Form7Received.Page" hidden-text="form 7 received" />
+        @if (Model.Form7Received is not null && Model.Form7Received.Equals("Yes"))
```

Comment:
This can be simplified with the `if` attribute by adding a convenience property on the model to encapsulate this, such as `if="@Model.WasForm7Received`.

Reply:
Done